app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,7 @@
         <activity android:name=".ListSalesActivity"
             android:parentActivityName=".MainActivity"></activity>
         <activity android:name=".LoginActivity"></activity>
+        <activity android:name=".UserProfileActivity"></activity>

```

Comment:
Same thing I told Sophie: You want to specify a parentActivity for all your activities but main.